### ISSUE TITLE
non-existent skills?

### ISSUE THREAD

**Issue by menguzat** [jgravelle/AutoGroq#13]
Hi, 
I used autogroq to create autogen agents and downloaded the json files.
There are files for agents, and the workflow. however, the agent files have skills that don't exist, and there are no skill definition files in the files I downloaded.

Is this a bug? if not, how do I add the skills?

`{
  "type": "assistant",
  "config": {
    "name": "codebase_analyst",
    "llm_config": {
      "config_list": [
        {
          "model": "gpt-4"
        }
      ],
      "temperature": 0.1,
      "timeout": 600,
      "cache_seed": 42
    },
    "human_input_mode": "NEVER",
    "max_consecutive_auto_reply": 8,
    "system_message": "You are a helpful assistant that can act as Codebase Analyst who Analyzes local codebases, identifies areas for improvement, and defines refactoring strategies."
  },
  "description": "Analyzes local codebases, identifies areas for improvement, and defines refactoring strategies",
  "skills": [
    "code_analysis",
    "software_architecture",
    "refactoring_techniques"
  ],
  "tools": [
    "sonarqube",
    "codeheat",
    "codemetrics"
  ]
}`

when I try to add this through autogenstudio, I get an "unprocessable entity" error.

**Comment by jgravelle** [comment]
Correct.  Those high-level descriptions currently serve as placeholders in
the JSON.  If you click the 'X's in AutoGen Studio to kill the empty
skills, your agent will load.

Thanks for checking it out!  We'll have some fundamental skill ability
added this week...


-jjg

**Comment by menguzat** [comment]
Oh cool!
I gathered that I could remove the skills, but the skills also seem useful so it would be really wonderful to have them generated also.

Great work btw! thanks!

**Comment by jgravelle** [comment]
Thank YOU...!
